app/uk/gov/hmrc/securemessage/controllers/model/cdcm/write/CdcmConversation.scala

```diff
@@ -47,7 +46,7 @@ final case class CdcmConversation(
       client,
       conversationId,
       ConversationStatus.Open,
-      Some(Map("mrn" -> tags.mrn, "notificationType" -> tags.notificationType.entryName)), //TODO: remove option as empty map can take place of that
+      Some(Map("mrn" -> tags.mrn, "notificationType" -> tags.notificationType.entryName)),
```

Comment:
Tags: Does it need to be an option?

Reply:
i wanted to remove option, but is is forced on us by play json, if we want to allow tag to be missing in the json payload, it need to either be Option or to have a custom serializer. Was a bit too much work for the later option, but happy if someone is brave enough to do this in a separate PR :).